src/components/pages/FavoriteItemsScreen.tsx

```diff
@@ -11,21 +10,25 @@ import {
 } from '../../config/selectors';
 import { BUILDER } from '../../langs/constants';
 import ErrorAlert from '../common/ErrorAlert';
+import { useFilterItemsContext } from '../context/FilterItemsContext';
 import ItemHeader from '../item/header/ItemHeader';
 import Items from '../main/Items';
 
 const FavoriteItemsLoadableContent = (): JSX.Element | null => {
   const { t: translateBuilder } = useBuilderTranslation();
   const { data, isLoading: isItemsLoading, isError } = hooks.useFavoriteItems();
+  const { shouldDisplayItem } = useFilterItemsContext();
+  // TODO: implement filter in the hooks directly ?
+  const filteredData = data?.filter((d) => shouldDisplayItem(d.item.type));
 
-  if (data) {
+  if (filteredData) {
```

Comment:
Do you display the select in this view ?

Reply:
Yes, it is not wanted ?

Reply:
It was not clear to me that the select was displayed here, all good then !